.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,19 @@
-<!--
 Thanks for submitting a PR, your contribution is really appreciated!
 
 Here is a quick checklist that should be present in PRs.
-(please delete this text from the final description, this is just a guideline)
--->
 
-- [ ] Target the `master` branch for bug fixes, documentation updates and trivial changes.
-- [ ] Target the `features` branch for new features, improvements, and removals/deprecations.
+!! Please delete this text from the final description, this is just a guideline) !!
+
+- [ ] Target the `master` branch for bug fixes, documentation updates and
+      trivial changes.  But please use `features` for changes touching many
```

Comment:
isn't the opposite slightly better (since we frequently merge master into features but rarely the other way)

Reply:
The point here is that it should go to the leading branch, and not the stable/release branch (for bugfixes).

Reply:
@asottile 
Can't you see / understand that?